src/core/css/inc/mixins.scss

```diff
@@ -385,4 +386,8 @@ $default-arrow-margin: 3px;
   background: $base-color;
   overflow: hidden;
   padding: 10px 20px;
+
+  @include respond-to(large) {
+    padding: 15px 36px;
```

Comment:
where is 15px used? Should it be 12px since that's a value used more frequently in the specs?

Reply:
If I go for `12px` then I'll have to change the `height` of the header card. `15px` is the value I read in Sketch (padding-bottom of the title in a header card) and is also used to vertically align the title.